v2/piv/key_go120.go

```diff
@@ -0,0 +1,115 @@
+//go:build go1.20
+// +build go1.20
+
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package piv
+
+import (
+	"crypto"
+	"crypto/ecdh"
+	"fmt"
+)
+
+type X25519PrivateKey struct {
```

Comment:
Please add a comment to this exported type similar to ECDSAPrivateKey, explaining that users are expected to type case private keys to this type. 

https://pkg.go.dev/github.com/go-piv/piv-go/v2/piv#ECDSAPrivateKey

Reply:
Fixed with cfec690